packages/main/src/Avatar.hbs

```diff
@@ -9,6 +9,7 @@
 	@click={{_onclick}}
 	role="{{_role}}"
 	aria-haspopup="{{_ariaHasPopup}}"
+	aria-label="{{accessibleNameText}}"
 >
 	{{#if hasImage}}
 		<slot></slot>
```

Comment:
Since the `accessibleNameText` is set to the root of `ui5-avatar` do we still need to provide it to the `ui5-icon`?